egs/librimix/ConvTasNet/local/conf.yml

```diff
@@ -0,0 +1,33 @@
+# filterbank config
+filterbank:
+  n_filters: 512
+  kernel_size: 16
+  stride: 8
+# Network config
+masknet:
+  n_blocks: 8
+  n_repeats: 3
+  mask_act: relu
+  bn_chan: 128
+  skip_chan: 128
+  hid_chan: 512
+# Training config
+training:
+  epochs: 8
```

Comment:
We might want to change that to something like 200

Reply:
Yeah sounds better ahah